products/statement-generator/src/components/AffirmationComponent.tsx

```diff
@@ -109,7 +109,9 @@ const AffirmationComponent = ({
             style={{
               display: 'flex',
               justifyContent: 'flex-end',
-              marginTop: '39px',
+              marginTop: '240px',
+              width: '700px',
+              paddingRight: '30px',
```

Comment:
Oof this component is old 😅  
It needs to be updated to use the `contentContainer` style to be uniform with the rest of the app I imagine https://github.com/hackforla/expunge-assist/blob/master/products/statement-generator/src/styles/utilityStyles.ts#L30-L33